locales/cy.json

```diff
@@ -0,0 +1,154 @@
+		"cancelButton": "Canslo’r taliad",
+		"goBackToService": "Mynd yn ôl i'r gwasanaeth"
+	},
+	"confirmDetails": {
+		"title": "Cadarnhau eich taliad",
+		"card": "Rhif cerdyn:",
+		"expiry": "Dyddiad dod i ben:",
+		"name": "Enw ar y cerdyn:",
+		"address": "Cyfeiriad bilio:",
+		"email": "E-bost cadarnhad:"
+	},
+	"commonConjunctions": {
+		"or": "neu"
+	},
+	"commonHeadings": {
+		"inProgress": "Mae eich taliad ar y gweill"
+	},
+	"paymentSummary": {
+		"title": "Crynodeb y taliad",
+		"totalAmount": "Cyfanswm:"
+	},
+	"errorViews": {
+		"either": "Gallwch:",
+		"continue": "Parhau gyda’ch taliad",
+		"tryAgain": "Mynd yn ôl i geisio’r taliad eto",
+		"problem": "Roedd problem gyda'ch taliad",
+		"declined": "Mae eich taliad wedi cael ei wrthod",
+		"contactYourBank": "Cysylltwch â'ch banc am ragor o fanylion. Nid oes arian wedi ei gymryd o'ch cyfrif",
+		"paymentStatus": "Roedd eich taliad yn",
+		"unsuccessful": "Nid oedd eich taliad yn llwyddiannus",
+		"noMoneyTaken": "Nid oes arian wedi ei gymryd o'r cyfrif.",
+		"toMakeChanges": "I wneud unrhyw newidiadau i'ch taliad mae angen i chi gysylltu â'r gwasanaeth.",
+		"viewSummary": "Gweld crynodeb eich taliad",
+		"paymentExpired": "Daeth eich taliad i ben",
+		"sessionExpired": "Daeth eich sesiwn taliad i ben",
+		"paymentCancelled": "Mae eich taliad wedi cael ei ganslo",
+		"paymentCancelledTitle": "Y taliad wedi ei ganslo",
+		"paymentCancelledDescription": "Mae eich taliad wedi ei ganslo gan y gwasanaeth llywodraeth. Gallai hyn fod wedi digwydd am nifer o resymau",
+		"error": "Digwyddodd gwall",
+		"technicalProblems": "Mae'n flin gennym ond rydym yn profi problemau technegol"
+	},
+	"authorisation": {
+		"extraSteps": "Efallai bydd angen i chi gwblhau camau gwirio pellach",
+		"approvalNeeded": "I barhau â'ch taliad, mae arnom angen cymeradwyaeth eich banc.",
+		"approved": "Mae manylion y cerdyn wedi ei gymeradwyo gan y banc.",
+		"checkingDetails": "Yn gwirio manylion y cerdyn",
+		"checkingDetailsDescription": "Mae eich manylion talu yn cael eu gwirio.",
+		"spinnerAltText": "yn aros",
+		"processingPayment": "Mae eich taliad yn cael ei brosesu.",
+		"processing": "Prosesu manylion y cerdyn",
+		"userCancelledPayment": "Mae eich taliad wedi ei ganslo"
+	},
+	"fieldErrors": {
+		"summary": "Mae’r meysydd canlynol yn wag neu yn cynnwys camgymeriadau",
+		"generic": "Rhowch %s dilys",
+		"fields": {
+			"cardholderName": {
+				"name": "enw",
+				"message": "Rhowch yr enw fel y mae'n ymddangos ar y cerdyn",
+				"containsTooManyDigits": "Rhowch yr enw fel y mae'n ymddangos ar y cerdyn",
+				"containsSuspectedCVC": "Rhowch yr enw fel y mae'n ymddangos ar y cerdyn"
+			},
+			"cardNo": {
+				"name": "rif cerdyn",
+				"message": "Rhowch rif cerdyn dilys",
+				"numberIncorrectLength": "Nid yw rhif y cerdyn yr hyd cywir",
+				"luhnInvalid": "Rhowch rif cerdyn dilys",
+				"cardNotSupported": "Nid yw'r math hwn o gerdyn yn cael ei dderbyn",
+				"nonNumeric": " Rhowch rif cerdyn dilys"
```

Comment:
Not sure about the leading space here but we I note we have one in the English version.